pkg/capacity/capacity_test.go

```diff
@@ -1217,7 +1264,8 @@ func fakeController(ctx context.Context, client *fakeclientset.Clientset, storag
 		driverName,
 		client,
 		queue,
-		defaultOwner,
+		&defaultOwner,
```

Comment:
Should there be a test case for "managed by us but not owned by us"?

Reply:
There is "reuse one capacity object, add owner" for that.

Reply:
The inverse "reuse one capacity object, remove owner" doesn't work because the driver instance without owner configured doesn't know which owner references it is expected to remove. This shouldn't be needed in practice and the documentation mentions it.

Reply:
I was actually referring to a test that captures the container-less driver case, where ownerref stays nil always. IIUC in all the tests so far assumes that capacity objects are eventually managed by a driver in containers (i.e. owner is always set to `defaultOwner`).

If you agree this test is necessary I would prefer that the test is added in this PR rather than delaying it after the release, because it is a use case we plan to support. Hopefully it's not a lot of work.

Reply:
You are right, that is a supported use case and wasn't covered. I've added two test cases for it which cover the relevant difference (create objects without owner reference, keep existing references).